avm/ptn/authorization/policy-exemption/main.bicep

```diff
@@ -0,0 +1,150 @@
+metadata name = 'Policy Exemptions (All scopes)'
```

Comment:
Is it okay to go with this approach for now and then refactor in the future? Reason being that I would like to use this for the refactor of ALZ-Bicep which is currently in progress.

Reply:
It is, yes. Just consider it a word of warnung, I guess 😄 

The transition would then look like this
1. This module is published
2. When the scoped-modules CI feature is live, we can get started on splitting the module into 3 submodules (if desired)
3. Once published, we'd unlist the original module (note, the module will still continue to work and not removed. It would just not be showing in the 'available modules' dropdown)